### PR TITLE
Add myself to the Tools team

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,6 +316,7 @@ The tools team maintains and develops core embedded tools.
 #### Members
 
 - [@adamgreig]
+- [@burrbull]
```
Comment (Disasm) [inline README.md]: Please, add a link to your profile (somewhere in this file)

Comment (burrbull) [reply README.md]: Done

Comment (Disasm) [reply README.md]: Thanks! Slightly not in alphabetical order, but works for me :)

**Comment by therealprof** [comment]
@rust-embedded/tools Can we get some thumbs up here?

**Comment by therealprof** [comment]
And we have majority. Welcome to the team @burrbull.

**Pull request merged**